docs/examples/301/web-app-managed-identity-sql-db/bicepsamples.csv

```diff
@@ -0,0 +1,142 @@
+"Name","BicepPath","BestPracticeResult","FairfaxDeployment","PublicDeployment","Status","QuickStartUri","BicepUri","QuickStartMoved","QuickStartAuthor","BicepAuthor","IsBicepQuickStart"
+"000/01-hello-world","docs/examples/000/01-hello-world",,,,,,,,,,
```

Comment:
>01 [](http://example.com/codeflow?start=5&length=2)

What's this file and the xls?

Reply:
those aren't supposed to be there, thanks.